iroh-unixfs/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "iroh-unixfs"
+version = "0.1.3"
+edition = "2021"
+authors = ["dignifiedquire <me@dignifiedquire.com>"]
+license = "Apache-2.0/MIT"
+repository = "https://github.com/n0-computer/iroh"
+description = "Implementation of unixfs for iroh"
+rust-version = "1.63"
+
+exclude = [
+  "tests/**",
+  "fixtures/**",
```

Comment:
not needed, as the `fixtures` are still in the resolver crate, right?

Reply:
hmmm, neither of these actually exists in this crate.

We do have a `benches` folder, should we be excluding that? Not sure what is "typically" published.

Reply:
no not needed to be excluded, we only have this because there are some larger fixtures